module/documents/activity/mixin.mjs

```diff
@@ -905,7 +915,11 @@ export default Base => class extends PseudoDocumentMixin(Base) {
     target.disabled = true;
     try {
       if ( handler ) await handler.call(activity, event, target, message);
-      else await activity._onChatAction(event, target);
+      else if ( action === "consumeResource" ) await this.#consumeResource(event, target, message);
+      else if ( action === "placeTemplate" ) await this.#placeTemplate();
```

Comment:
Shouldn't these be in `metadata.usage.actions`?

Reply:
Can't because with the Mixin there isn't a concrete class to reference.